src/components/TopBar/visa-status-dropdown.js

```diff
@@ -37,10 +37,14 @@ export default class VisaStatusDropdown extends React.Component {
       subheaderText = visaTypes.visible.length + " Selected";
     }
 
-    const preferredVisaTypes = viewAllOptions
-      ? visaTypes.allVisas
-      : visaTypes.allVisas.slice(0, 1);
+    const preferredVisaTypes = visaTypes.allVisas.length > 3 ?
+      (viewAllOptions
```

Comment:
.slice can take an "end" beyond the actual length so you don't need the length check

Reply:
Additional suggestion, make the literal "3" into a constant either in the file or in a JSON file (I think we have some JSON config files for constants under the assets folder)

Reply:
+1 to a local for the 3, so it'll be easier to change later

Reply:
Changed the 3 to part of the state for the component and remove the unnecessary length check